content/rdi/monitoring-guide.md

```diff
@@ -5,103 +5,68 @@ description: Monitor RDI Engine and data processing jobs
 weight: 70
 alwaysopen: false
 categories: ["redis-di"]
-aliases: 
+aliases:
 ---
 
-RDI Engine accumulates operating statistics that you can: 
+RDI Engine accumulates operating statistics that you can:
 
-* Observe and analyze to discover various types of problems.
-* Use for optimization purposes.
+- Observe and analyze to discover various types of problems.
+- Use for optimization purposes.
 
 ## Console metrics
 
-Some basic RDI operating metrics can be displayed using the [`redis-di status`]({{<relref "/rdi/reference/cli/redis-di-status">}}) command. The command provides information about the current RDI Engine status, target database configuration and processing statistics broken down by stream. This tool is intended to be used by an Operator to get the current snapshot of the system.
+RDI can display its operating metrics in the console using the [`redis-di status`]({{<relref "/rdi/reference/cli/redis-di-status">}}) command. The command provides information about the current RDI Engine status, target database configuration and processing statistics broken down by stream. This tool is intended to be used by an Operator to get the current snapshot of the system as well as the ongoing data processing monitoring (when used in live mode).
 
 ## Prometheus integration
 
-RDI allows exporting its metrics to [Prometheus](https://prometheus.io/) and visualizing them in [Grafana](https://grafana.com/). Currently, RDI relies on the external [OSS Redis Exporter](https://github.com/oliver006/redis_exporter) that connects to RDI database to source the metrics and serve them for Prometheus job scraping. The diagram below describes this flow and components involved.
+RDI allows collecting and exporting its metrics to [Prometheus](https://prometheus.io/) and visualizing them in [Grafana](https://grafana.com/). Operator can start the built-in exporter using the [`redis-di monitor`]({{<relref "/rdi/reference/cli/redis-di-monitor">}}) command. The diagram below describes this flow and components involved.
```

Comment:
Can be in a different line with this text: 
This diagram describes this flow and components involved:

Reply:
fixed